src/edu/mills/cs180a/VisibleForTesting.java

```diff
@@ -8,5 +8,5 @@
  * visibility than otherwise needed to enhance testability.
  */
 @Retention(SOURCE)
-public @interface VisibleForTesting {	
+public @interface VisibleForTesting {		
```

Comment:
There are some extra spaces at the end of the line. This will be fixed by the formatter.